### PR TITLE
Add August update

### PR DESCRIPTION
**Pull request by pablogsal**

### CODE REVIEW

**Comment by Yhg1s** [comment]
LGTM but you need to run cog to add it to the main README.md file.

**Comment by pablogsal** [comment]
TIL that installing `cog` installs a wheel for `grpcio`!

**Comment by pablogsal** [comment]
Ah, wrong `cog`, is `cogapp` 😓

File: updates/2022-08-steering-council-update.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  https://peps.python.org/pep-0659/). The SC decided that for [PEP 659–
+  Specializing Adaptive Interpreter](https://peps.python.org/pep-0659/)  in
+  particular the status should be Final.
+* The SC agreed that we want to avoid having people write a PEP after pushing
+  through an implementation.
+* The SC discussed CPython's backward compatibility policy and decided that we
+  should gather input from the core devs on that. It was proposed to ask people
+  attending the core dev sprint as a first measure.
+
+## 2022-08-22
```
Comment (merwok) [inline updates/2022-08-steering-council-update.md:89]: Same date as previous heading

Comment (encukou) [reply updates/2022-08-steering-council-update.md:89]: Thanks! Fixing in https://github.com/python/steering-council/pull/149

**Pull request merged**